build.gradle

```diff
@@ -90,7 +90,7 @@ subprojects {
 	
 	            pom {
 	                groupId = project.group
-	                artifactId = project.archivesBaseName
+	                artifactId = project.name
```

Comment:
Perfect. This should simplify the setup. Can you also remove the ``archivesBaseName`` config setting in the subproject build scripts? It is no longer needed.